### ISSUE TITLE
Add info about signature formatting

### ISSUE THREAD

**Issue by paluh** [lambdaterms/purescript-style-guide#2]
We prefer format which is proposed by garyb here: https://github.com/ianbollinger/purescript-style-guide/issues/4

**Comment by paluh** [comment]
Function:
```purescript
logShow
  ∷ ∀ a eff
  . Show a 
  ⇒ a 
  → Eff (console :: CONSOLE | eff) Unit
logShow a = log (show a)
```

Typeclasses:

```purescript
instance consCols                                                                                                                                                   
  ∷ ( QueryCols s tail r'
      , IsSymbol name                                                                                                                                                  
      , RowLacks name r'
      , RowCons name (Col s a) r' r
      )
  ⇒ QueryCols s (Cons name a tail) r
  where
  colsImpl _ = do
      ...
```

**Comment by paluh** [comment]
Fixed: e579fc813b58fa2f2f693faea7836dd7eddf5201
